app/helpers/questions_helper.rb

```diff
@@ -0,0 +1,25 @@
+module QuestionsHelper
+  SORTING_OPTIONS = %w(All Week Month Year).freeze
+
+  def filtering(period)
```

Comment:
This looks good but perhaps it could have a more descriptive name -- something describing how it's related to stats, or Q&A? Thanks!

Reply:
Also, I'd love to see a simple test for these methods. Where would that live?

Reply:
Let me try and figure out where the tests will go. Thanks